tests/Tests.AzureAppConfiguration/LoggingTests.cs

```diff
@@ -308,6 +567,21 @@ private bool ValidateLoggedError(Mock<ILogger> logger, string expectedMessage)
             return true;
         }
 
+        private bool ValidateLoggedSuccess(Mock<ILogger> logger, string expectedMessage)
```

Comment:
Can you also add tests to check if the key names are properly logged in debug logger?